### ISSUE TITLE
Touch screen on chromebook

### ISSUE THREAD

**Issue by rcvangemert** [microsoft/vscode#98218]
Issue Type: <b>Feature Request</b>

Can we have touch screen scrolling on chromebooks. I understand virtual keyboard may not be possible yet but basic cursor movement and scrolling should be possible.

VS Code version: Code 1.44.2 (ff915844119ce9485abfe8aa9076ec76b5300ddd, 2020-04-16T17:50:03.709Z)
OS version: Linux x64 4.19.113-08528-g5803a1c7e9f9

**Comment by abhijitchikane** [comment]
Please just allow to scroll with the touched for now

Thank you

**Comment by acerspyro** [comment]
Fedora 36, GNOME 42

Can't drag the window around when using a "custom" title bar, and I would love to be able to just scroll around windows by using two fingers, and highlight text using one finger.

**Comment by CrocsAreComfy** [comment]
Can we get this added? Really frustrating to not be able to scroll by touch when referencing code and collaborating around the office.

**Comment by acerspyro** [comment]
Can't argue; Crocs ARE comfy!

Does this feature work on Windows? The only touchscreen laptop I have runs Linux.

**Comment by jorgemop** [comment]
Is Anyone Here Alive?

**Comment by rcvangemert** [comment]
I am!

**Comment by adri326** [comment]
It seems like some of the parts of vscode work just fine with touchscreen, like the terminal or the settings window, but others don't, and that includes the actual editor area

**Comment by trustytrojan** [comment]
> Does this feature work on Windows? The only touchscreen laptop I have runs Linux.

yes, it works on windows just fine, but not linux, on either wayland or X. in fact X just treats touchscreens like regular old mice, so trying to scroll just starts selecting text :skull: 

scrolling in every other pane in wayland works except the editor, when will this get implemented :sob:

**Comment by d-bucur** [comment]
On Linux you can add the **--touch-events** argument when executing code ie
`code . --touch-events`
and touch scroll will work the same way it does in windows by default. Maybe you can do the same on Chromebook
[source](https://www.reddit.com/r/GPDPocket/comments/ajbqrt/wayland_made_vscode_touch_scroll_stop_working/)

**Comment by d-bucur** [comment]
Adding some more info on the issue.

The web version deployed at https://vscode.dev/ and https://github.dev doesn't handle touch events properly inside the editor area for code, for example when opening a .py file, but does work correctly when opening read only files, like the preview for an .md file.

So unfortunately I can't use touch drag scroll on my 2in1 Lenovo and browsing some github codebase, which would be super useful.

System info: Chrome 137.0.7151.55 on Ubuntu 22.04. Same thing happens under Firefox 138

Edit: Adding the mighty `--touch-events` command line switch when launching Chrome also fixes the issue. I am not even sure why this is not enabled by default. More info on chrome command line switches [here](https://gist.github.com/dodying/34ea4760a699b47825a766051f47d43b).
